data/iso4217/amendments/168.json

```diff
@@ -0,0 +1,628 @@
+[
```

Comment:
I put this file here because I couldn't figure out how to import a json directly from a package, so I just put it in the top level.